.github/workflows/build-and-test.yml

```diff
@@ -125,7 +125,11 @@ jobs:
 
       - name: Start iota sandbox
         if: matrix.os == 'ubuntu-latest'
-        uses: './.github/actions/iota-sandbox/setup'
+        uses: 'iotaledger/iota-sandbox/.github/actions/setup@add-action'
+        with:
+          branch: add-action
+        env:
+          COMPOSE_PROFILES: 'inx-faucet.inx-indexer'
```

Comment:
```suggestion
          COMPOSE_PROFILES: 'inx-faucet,inx-indexer'
```

Reply:
Btw you could also just set env once for the whole file. Less room for error

Reply:
I was actually thinking about wrapping the sandbox action in our own action for parametrization